src/BlazorUI/Bit.BlazorUI/Components/Inputs/TagsInput/BitTagsInput.scss

```
@@ -0,0 +1,141 @@
+@import "../../../Styles/functions.scss";
```

Stylelint (17.7.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)